### PR TITLE
Several new features for the 0.15 release

### PR DESCRIPTION
**Pull request by pcenov**

### CODE REVIEW

File: integration_testing/features/super-admin/super-admin-import-individual-users.feature [hunk 1]
```diff
@@ -0,0 +1,152 @@
+Feature: Import individual users
```
Comment (radinamatic) [inline integration_testing/features/super-admin/super-admin-import-individual-users.feature]: Let's just change the filename to reflect that it's a part of the setup wizard procedure:
`super-admin-setup-wizard-import-individual-users.feature`

File: integration_testing/features/super-admin/super-admin-import-individual-users.feature [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+			And I open Kolibri in my browser
+		Then I see *Please select the default language for Kolibri*
+		When I click *Continue*
+		Then I see *Getting started*
+		When I select *Advanced Setup*
+			And I click *Continue*
+		Then I see *Device name*
+			And I see <My device's name>
+		When I click *Continue*
+		Then I see *Initialize device*
```
Comment (radinamatic) [inline integration_testing/features/super-admin/super-admin-import-individual-users.feature]: We changed the wording here, check the latest on the Figma prototype:
https://www.figma.com/file/w3raYcGRUhhaVm1Of8Dcvo?node-id=229:63#94924868

Comment (pcenov) [reply integration_testing/features/super-admin/super-admin-import-individual-users.feature]: Ok, I've changed 'Initialize device' to 'Select a facility setup for this device' here and in subsequent lines.

Comment (radinamatic) [reply integration_testing/features/super-admin/super-admin-import-individual-users.feature]: Good, push both changes in a new commit and I'll approve! 👍🏽

**Review by radinamatic — APPROVED** [review]
First 0.15 season pickles, 😋!

**Pull request merged**